src/olympia/amo/admin.py

```diff
@@ -642,6 +645,27 @@ def choices(self, changelist):
         yield all_choice
 
 
+class NumericRangeFilter(FakeChoicesMixin, NumericRangeFilterBase):
+    """
+    Custom rangefilter.filters.NumericRangeFilter class without the need for
+    inline CSS/JavaScript.
+
+    Needs FakeChoicesMixin for the fake choices the template will be using (the
+    upstream implementation depends on inline JavaScript for this, which we
+    want to avoid).
+    """
+
+    template = 'admin/amo/numeric_range_filter.html'
+
```

Comment:
When testing the label tags were empty (the label tag existed, but there was no text value). Date range filter fields get "To" and "From" labels; numeric range filter fields did not - though the placeholder texts did say "To" and "From".

Do you need to override `_get_form_fields` too?

At that point, you probably want to inherit from DateRangeFilter, or a common ancestor - it's worth considering anyway as your `choices` seems to be a direct copy and paste.